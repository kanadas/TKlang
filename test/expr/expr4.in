if !false then 1 @ 'a' : "la ma" ++ [' ', 'k', 'o', 't', 'a'] else 2 @ ()
