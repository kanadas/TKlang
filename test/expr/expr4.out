expr/expr4.in
Error: Undefined case: EUnion (EInt 2) EVoid

[Abstract Syntax]

EIf (ENot ETrue) (EUnion (EInt 1) (EAppend (EChar 'a') (EConcat (EString "la ma") (EList [EChar ' ',EChar 'k',EChar 'o',EChar 't',EChar 'a'])))) (EUnion (EInt 2) EVoid)

[Linearized tree]

if ! true then 1 @ 'a' : "la ma" ++ [' ', 'k', 'o', 't', 'a'] else 2 @ ()
